src/gsp.py

```diff
@@ -251,11 +271,18 @@ def get_truths_for_all_gsps(
     - **start_datetime_utc**: optional start datetime for the query.
     - **end_datetime_utc**: optional end datetime for the query.
     """
-    logger.info(f"Get PV Live estimates values for all gsp id and regime {regime} for user {user}")
 
-    if gsp_ids is not None:
+    if isinstance(gsp_ids, str):
         gsp_ids = [int(gsp_id) for gsp_id in gsp_ids.split(",")]
 
+    if is_fake:
+        if gsp_ids is None:
+            gsp_ids = [List[int(gsp_id)] for gsp_id in range(1, GSP_TOTAL)]
```

Comment:
should this be `gsp_ids = [int(gsp_id) for gsp_id in range(1, GSP_TOTAL)]`

Reply:
Awesome! Thanks for this. Looks like it's worked!